readme.md

```diff
@@ -39,6 +39,7 @@ Feel free to submit a PR adding a link to your own recaps, guides or reviews!
 * [Abid Hasan's](https://github.com/sabidhasan/javascript-30) completion of all challenges. This was awesome!! Learned so much! :+1:
 * [Yusong Notes](https://sky172839465.github.io/course/js30) Records Yusong JS 30 days note and demo :star2:
 * [Herminio Torres](https://github.com/herminiotorres/JavaScript30) lessons and tricks learned, and a [gh-page](https://herminiotorres.github.io/JavaScript30/) to see working all the mini-projects.
+* [Thomas Mattacchione](https://github.com/tkjone/clojurescript-30) JavaScript 30 written in ClojureScript
```

Comment:
Hi, thanks for your contribution. Just wanted to know, is there any way to run these demos online without needing to setup anything like others have done using [GitHub pages](https://misslild.github.io/WesBos-30day-Coding-challenge/) or [codepen](https://codepen.io/collection/DapZeP/) or others?

Regards!